api/src/main/resources/workbench-api.yaml

```diff
@@ -350,6 +350,37 @@ paths:
           description: Bad request
           schema:
             "$ref": "#/definitions/ErrorResponse"
+  "/v1/link-ras-account":
+    post:
+      tags:
+      - profile
+      consumes:
+      - application/json
+      description: Link user with a RAS account
+      operationId: linkRasAccount
+      parameters:
+      - in: body
+        name: authcode
+        description: The authorization code from RAS.
+        schema:
+          "$ref": "#/definitions/RasLinkRequestBody"
+      responses:
+        200:
+          description: The user's account linking information.
+          schema:
+            "$ref": "#/definitions/AccountLinkInfo"
+        400:
+          description: Bad request
+          schema:
+            "$ref": "#/definitions/ErrorResponse"
+        500:
+          description: Internal Error
+          schema:
+            "$ref": "#/definitions/ErrorResponse"
+        501:
```

Comment:
We don't generally use 501 so it's possible that our UI code would not handle this well.  Not sure.

Reply:
That is an general code for the features that does not supported yet. I probably will just remove that in my next PR when I work on the actual endpoint implementation? Or you want me to remove it from swagger now/ 
And for this case, I don't think this UI code will hit this endpoint, highly possible the call from UI is feature protected.
Fine to remove it now, wdyt?

Reply:
OK to keep it.  501 makes sense.  Just a comment that it's something to look out for.

Reply:
Alternatively, you can just drop it. I don't think this has a direct impact on the codegen aside from documentation, except if you want to embed a model in there, e.g. `ErrorResponse`; we can still throw a 501 without documenting that return code.